src/status.rs

```diff
@@ -255,6 +337,42 @@ mod test {
         channel
     }
 
+    fn get_approve_state_msg() -> ApproveStateValidatorMessage {
+        ApproveStateValidatorMessage {
+            from: DUMMY_VALIDATOR_LEADER.id,
+            received: Utc::now(),
+            msg: MessageTypes::ApproveState(ApproveState {
+                state_root: String::from("0x0"),
+                signature: String::from("0x0"),
+                is_healthy: true,
+            }),
+        }
+    }
+
+    fn get_heartbeat_msgs(recency: Duration) -> HeartbeatValidatorMessage {
+        HeartbeatValidatorMessage {
+            from: DUMMY_VALIDATOR_LEADER.id,
```

Comment:
can we also pass the `from: ValidatorId` it should make some tests simpler to write instead of overriding the `from`